src/components/modals/TransactionReceiptModal.tsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import SuccessIcon from "../../assets/svgs/success-icon.svg";
+import CloseIcon from "../../assets/svgs/close-circle.svg";
+import HyveIcon2 from "../../assets/svgs/hyve-icon2.svg";
+import hyvePay from "../../assets/images/hyvePay.png";
+import { jsPDF } from "jspdf";
+import html2canvas from "html2canvas";
+import useAppSelector from "../../hooks/useAppSelector";
+import moment from "moment";
+import { useUser } from "../../hooks/useUser";
+import { Util } from "../../helpers/Util"
+
+//FIXME: fix this by making .env to work on this codebase.. 
+const TRANSFER_FEE = 10
```

Comment:
This should be an .env variable

Reply:
when i use import meta.env as used before it does not work i tried to use process.env it didn't work too because the react-script is no longer there. so i used this for the main time while i look for a solution to resolve the use of env

Reply:
You need to add VITE_ before the constant name